tooling/nargo_cli/tests/compile_success_empty/raw_string/src/main.nr

```diff
@@ -0,0 +1,10 @@
+global D = r#####"Hello "world""#####;
+
+fn main() {
+    let a = "Hello \"world\"";
+    let b = r#"Hello "world""#;
+    let c = r##"Hello "world""##;
+    assert(a == b);
```

Comment:
can you check that  `"r#"Hello World"#"` works as well?

Reply:
Do you mean, `r##"r#"Hello World"#"##` or `r#"Hello World"#`?

Reply:
I added `r#"Hello World"#` in case you meant the most basic case.

Reply:
my bad, forgot the escapes.

i meant checking that a string with a raw-string specifier is still parsed as a string
`"r#\"Hello World\"#"`

Reply:
@grasshopper47 there's no need for such a case, because of how the lexer works: it simply goes from left to right while committing to whatever it finds. Here, the first character begins a string and whatever follows to the right will be seen as a string to every lexer.

Reply:
@grasshopper47 the most important tests are already added, but I added nested stuff similar to your idea

Reply:
Done.

Reply:
> @grasshopper47 there's no need for such a case, because of how the lexer works: it simply goes from left to right while committing to whatever it finds. Here, the first character begins a string and whatever follows to the right will be seen as a string to every lexer.

I know, but better safe than sorry, as the cost was minimal I thought.



> @grasshopper47 the most important tests are already added, but I added nested stuff similar to your idea

Good, at least we extended coverage, which was the initial desire anyway.

Nicely done!